recipes-kernel/linux/linux-yocto_6.10.bbappend

```diff
@@ -0,0 +1,32 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```

Comment:
Add the `:tegra` override here and that should take care of the issue being reported by `yocto-check-layer`.